packages/react-accordion/src/Accordion.stories.tsx

```diff
@@ -1,13 +1,25 @@
 import * as React from 'react';
+import { RocketIcon } from './icons.stories';
 import { AccordionItem, AccordionHeader, AccordionPanel, Accordion, AccordionProps } from './index';
 import { ArgType } from '@storybook/addons';
-import { RocketIcon } from './icons.stories';
+import { AccordionHeaderProps } from './components/AccordionHeader/AccordionHeader.types';
+
+interface AccordionExampleProps
+  extends AccordionProps,
+    Pick<AccordionHeaderProps, 'icon' | 'inline' | 'size' | 'expandIconPosition'> {}
 
-export const AccordionExample = (props: AccordionProps) => {
+export const AccordionExample = ({ icon, inline, size, expandIconPosition, ...props }: AccordionExampleProps) => {
```

Comment:
why is there an icon prop here ? is this expected to be passed from storybook controls ?

Reply:
Yes, the storybook will pass icon as a boolean, and this property is not from `Accordion`, that's why `AccordionExampleProps` was create, to differ what belongs to `Accordion` and what doesn't

Reply:
oh icon is a boolean 🤔🤔 I see